go/cmd/dolt/commands/clone.go

```diff
@@ -91,6 +93,11 @@ func Clone(ctx context.Context, commandStr string, args []string, dEnv *env.Dolt
 				if verr == nil {
 					verr = cloneRemote(ctx, srcDB, remoteName, branch, dEnv)
 
+					if verr == nil {
+						evt := events.GetEventFromContext(ctx)
+						evt.SetAttribute(eventsapi.AttributeID_CLONE_REMOTE_URL, remoteUrl)
+					}
+
```

Comment:
For `clone` command, my thinking is we probably want to see the url of the remote.